scripts/ci/ensure-staged-sass.sh

```diff
@@ -1,12 +1,28 @@
 #!/bin/bash
 
+###
+# Copyright 2017 resin.io
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
 set -e
 set -u
 
 npm run sass
 
 # From http://stackoverflow.com/a/9393642/1641422
-if [[ -n $(git status -s) ]]; then
+if [[ -n $(git status -s | grep "\\.css$" || true) ]]; then
```

Comment:
I started modifying the bash scripts yesterday, but got a bit carried away with making _all_ our bash scripts more uniform (and then got side-tracked with the separate CLI-path-escaping stuff), so feel free to merge this if the CI tests are passing, and I can PR my bash-changes later.